### PR TITLE
Help button addition

### PR DESCRIPTION
**Pull request by GuyDuerFB**
A new doc describing the help menu in the app.

### CODE REVIEW

File: docs/general-reference/help-button.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: Firebolt Help
+description: Describes the options of the Firebolt "Help" button
+nav_exclude: true
```
Comment (molly-firebolt) [inline docs/general-reference/help-button.md]: is excluding it from the nav (left menu) intentional?

File: docs/general-reference/help-button.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+![Help Menu](../assets/images/help_menu.png)
+
+
+### Status Page
+The status page allows you to view in real-time the operational status of our systems and descriptions of historical incidents. 
+
+You can also access the status page by browsing directly to it through https://firebolt.statuspage.io
+
+#### Main Page
```
Comment (molly-firebolt) [inline docs/general-reference/help-button.md:32]: intentional different formatting than status page? If not, I think you want ### not ####

File: docs/general-reference/help-button.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+2. **Text notifications** - by providing your phone number. 
+*Please note that text messages will only notify you when Firebolt creates or resolves an incident.*
+3. **Slack notifications** - by connecting via Slack.
+*Please note - you can also get maintenance status through Slack* 
+4. **RSS updates** - by connecting via your RSS/Atom application
+
+### Contact Support
+Contact Support allows you to create a support case for Firebolt's support team.
+
+![Create Case](../assets/images/create_support_case.png)
```
Comment (molly-firebolt) [inline docs/general-reference/help-button.md:80]: I would not use your real email in this image .... @example.net works great as a demo

**Review by molly-firebolt — CHANGES_REQUESTED** [review]

File: docs/general-reference/help-button.md (hunk 1 above)
Comment (GuyDuerFB) [reply docs/general-reference/help-button.md]: No, fixed

File: docs/general-reference/help-button.md (hunk 2 above)
Comment (GuyDuerFB) [reply docs/general-reference/help-button.md:32]: The main page is a part of the status page:
-- Status page
     | -- Main page 
     | -- Historical uptime

So the levels were intentional.

File: docs/general-reference/help-button.md (hunk 3 above)
Comment (GuyDuerFB) [reply docs/general-reference/help-button.md:80]: Done

**Comment by molly-firebolt** [comment]
@GuyDuerFB I'm going to pull this into the contributor repo so that we can preview it before publishing - contributor repo is preferred for big changes in the future, please, so that we can do this preview.

**Pull request closed without merging**